aderyn_driver/src/driver.rs

```diff
@@ -33,6 +33,13 @@ pub fn drive(args: Args) {
     drive_with(args, get_all_issue_detectors());
 }
 
+pub fn drive_highs_only(args: Args) {
```

Comment:
Does it make sense to introduce another drive function? It feels like this path should be covered in `drive_with`. What do you think?

Reply:
Yes. I think so too. Let me do that

Reply:
Done ✅ - Removed the extra function by including in `highs_only` as a field in `Args` 

Also included test to generate json report for highs-only